### PR TITLE
Milestone 4: Desktop Version

### PR DESCRIPTION
**Pull request by zohra-neda**
In this project we :
 - Add desktop version navbar menu
 - Add desktop version style
 - Uplode desktop version images
 - Update README.md file live demo link

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -134,7 +147,7 @@ <h2 class="cart-project-item cart-title">Professional Art Printing Data</h2>
     <!-- About section -->
     <section class="about" id="about">
         <div class="about-content">
-            <div class="about-items">
+            <div class="about-items des-about-item">
                 <h2 class="about-title">About me</h2>
                 <p class="about-p">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
                 <button class="resume-btn buttons">Get my resume</button>
```
Comment (Shedrack-Sunday) [inline index.html:153]: - [ ] Nice design on your about section. However, kindly align the button carefully as placed on the figma template given 👍 
- I've made a detailed comparison below, to show you the visual difference 👍 

## Your view 
image

## Figma:
image

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @zohra-neda           👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well implementing the requirements,  However, there are a few you need to work on, to move to the next project. 💯

 


## Highlights
✔️ Nice use of CSS for styling
✔️ Nice code implementation
✔️ Moderate attention to detail
  
# Required Changes ♻️

- There's a bit of styling that needs to be an adjustment 

_check the comment section for a detailed review_ 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by zohra-neda** [comment]
I fixed the button alignment on 1440 resolution

**Review by osamaashraf6 — APPROVED** [review]
Hi @zohra-neda 😊, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors present
- The PR has a proper title and a descriptive body 👍

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**